core-pasivos/src/main/java/com/banquito/corepasivos/client/model/RelationshipClient.java

```diff
@@ -0,0 +1,48 @@
+package com.banquito.corepasivos.client.model;
+
+import java.sql.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "RELATIONSHIP_CLIENT")
+public class RelationshipClient {
+
+    @EqualsAndHashCode.Include
+    @Id
+    @Column(name = "CODE_RELATIONSHIP", nullable = false )
+    public Integer codeRelationship;
+    
+    @ManyToOne
+    @JoinColumn(name = "CODE_CLIENT", referencedColumnName = "CODE_CLIENT", nullable = false)
+    public Client codeClient;
+
+    @ManyToOne
+    @JoinColumn(name = "CODE_CLIENT_RELATIONSHIP", referencedColumnName = "CODE_CLIENT", nullable = false)
```

Comment:
el `referencedColumnName` en este caso no seria `CODE_CLIENT_RELATIONSHIP` tambien?

Reply:
No porque como ambas llaves son foráneas hacen referencia a la clave primaria de la tabla cliente que se llama según el modelo CODE_CLIENT.